tracer/test/test-applications/integrations/Samples.VersionConflict.2x/Samples.VersionConflict.2x.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  
+  <ItemGroup>
```

Comment:
This may be a dumb question, but how does running a restore on version `2.255.251` result in us referencing a Datadog.Trace with assembly version `2.255.251.0`? Something's not clicking for me

Reply:
The last digit is always `0` e.g. currently, for version `1.30.1`, the assembly version is `1.30.1.0`:

https://github.com/DataDog/dd-trace-dotnet/blob/b8205b8b67a4a9f11ddc8ad0eec4d1d76fc78cf1/tracer/src/Datadog.Trace/TracerConstants.cs#L17

Reply:
I was more confused about how we build a 2.55.55 version, because we don't actually have a published NuGet package with that version, right?